theme/templates/base.html

```diff
@@ -25,6 +25,8 @@
         {% endif %}
         <!-- Mobile nav script -->
         <script type="text/javascript" src="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/scripts/mobile-nav.js"></script>
+        <!-- Image gallery handler script -->
+        <script type="text/javascript" src="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/scripts/img-gallery-handler.js"></script>
```

Comment:
This needs to be loaded only for the index page.

Where is the `img-gallery-handler.js` anyway ?